v19.2/performance-tuning.md

```diff
@@ -658,20 +658,16 @@ $ cockroach sql \
 --execute="CREATE INDEX ON rides (start_time) STORING (rider_id);"
 ~~~
 
-{{site.data.alerts.callout_info}}
-The `rides` table contains 1 million rows, so adding this index will take a few minutes.
-{{site.data.alerts.end}}
-
 Adding the secondary index reduced the query time from 1573ms to 61.56ms:
 
 {% include copy-clipboard.html %}
 ~~~ shell
 $ {{page.app}} \
 --host=<address of any node> \
---statement="SELECT count(DISTINCT users.id) \
-FROM users \
-INNER JOIN rides ON rides.rider_id = users.id \
-WHERE start_time BETWEEN '2018-07-20 00:00:00' AND '2018-07-21 00:00:00'" \
+--statement="SELECT count(DISTINCT users.id)
+FROM users
```

Comment:
Is the idea here that the `\` characters are no longer needed?  I did not test this but I assume it works fine?  Did it work fine before and you are removing the `\` chars for style reasons?  (I see the date update below of course).